src/test/scala/com/ibm/couchdb/api/DocumentsSpec.scala

```diff
@@ -218,6 +218,35 @@ class DocumentsSpec extends CouchDbSpecification {
       aliceRes2.doc.age mustEqual fixAlice.age + 1
     }
 
+    "Update a document with attachments" >> {
+      val created = awaitRight(documents.create(fixAlice))
+      val aliceRes = awaitRight(documents.get[FixPerson](created.id))
+      val aliceWithAttachments = aliceRes.copy(
+        _attachments = Map(
+          fixAttachmentName -> CouchAttachment.fromBytes(
+            fixAttachmentData, fixAttachmentContentType),
+          fixAttachment2Name -> CouchAttachment.fromBytes(
+            fixAttachment2Data, fixAttachment2ContentType)))
+      val docOk = awaitRight(documents.update(aliceWithAttachments))
+      checkDocOk(docOk, aliceRes._id)
+      val doc = awaitRight(documents.get.attachments().query[FixPerson](created.id))
+      doc._id mustEqual created.id
+      doc._attachments must haveLength(2)
```

Comment:
can we remove this if the second line is changed to `containTheSameElementsAs` or `contain(exactly(...))` matchers? Not sure if we need to update to specs2 first though